fundamentals/math.test.js

```diff
@@ -4,8 +4,14 @@
  */
 
 
-function gcd(a, b) {
-    return 0
+function gcd(a, b) {    
+    let arr = [];
+
+    for (let i = 0; i <= (a <= b ? a : b); i++) {

```

Comment:
walk me through this code. You create an array, fill it with numbers from 0 to a or b (whatever is smaller) and then you filter out to see which are good candidates. Then you get the biggest one.

It's correct, but can you spot anything to be more efficient?